packages/ubuntu_desktop_installer/lib/pages/turn_off_secure_boot/turn_off_secure_boot_page.dart

```diff
@@ -0,0 +1,130 @@
+          padding: const EdgeInsets.symmetric(horizontal: 20),
+          child: Form(
+            key: _formKey,
+            child: Column(
+              crossAxisAlignment: CrossAxisAlignment.stretch,
+              children: [
+                Align(
+                  alignment: Alignment.centerLeft,
+                  child: Text(lang.turnOffSecureBootDescription),
+                ),
+                RadioListTile<SecureBootMode>(
+                  title: Text(lang.turnOffSecureBootOption),
+                  value: SecureBootMode.turnoff,
+                  groupValue: model.secureBootMode,
+                  onChanged: model.setSecureBootMode,
+                ),
+                const SizedBox(height: _kFormElementPadding),
+                _PasswordSizedBox(
+                  child: TextField(
+                    obscureText: true,
+                    onChanged: (value) {
+                      model.setSecurityKey(value);
+                      _formKey.currentState?.validate();
+                    },
+                    decoration: InputDecoration(
+                        border: const OutlineInputBorder(),
+                        hintText: lang.chooseSecurityKey,
+                        enabled: model.areTextFieldEnabled),
+                  ),
+                ),
+                const SizedBox(height: _kFormElementPadding),
+                _PasswordSizedBox(
+                  child: TextFormField(
+                    obscureText: true,
+                    validator: (value) {
+                      if (!model.isConfrmationKeyValid) {
+                        return lang.secureBootPasswordsDontMatch;
+                      }
+                      return '';
+                    },
+                    onChanged: (value) {
+                      model.setConfirmKey(value);
+                      _formKey.currentState?.validate();
+                    },
+                    decoration: InputDecoration(
+                        border: const OutlineInputBorder(),
+                        hintText: lang.confirmSecurityKey,
+                        enabled: model.areTextFieldEnabled),
+                  ),
+                ),
+                const SizedBox(height: _kFormElementPadding),
+                RadioListTile<SecureBootMode>(
+                  title: Text(lang.dontInstallDriverSoftwareNow),
+                  subtitle: Text(lang.dontInstallDriverSoftwareNowDescription),
+                  value: SecureBootMode.dontinstall,
+                  groupValue: model.secureBootMode,
+                  onChanged: model.setSecureBootMode,
+                ),
+              ],
+            ),
+          ),
+        ),
+        actions: <WizardAction>[
+          WizardAction(
+            label: lang.backButtonText,
+            onActivated: Navigator.of(context).pop,
+          ),
+          WizardAction(
+            label: lang.continueButtonText,
+            onActivated: model.isFormValid
```

Comment:
`Wizard.of(context).next`, and feel free to use `enabled: model.isFormValid` to get rid of the condition in the callback, to make the code look (subjectively) nicer